src/js/features/actions/index.js

```diff
@@ -50,3 +65,17 @@ export const SET_TRANSLITERATION_LANGUAGES = 'SET_TRANSLITERATION_LANGUAGES';
 export const setTransliterationLanguages = createAction(
   SET_TRANSLITERATION_LANGUAGES
 );
+
+export const SET_DARK_MODE = 'SET_DARK_MODE';
+export const setDarkMode = createAction(SET_DARK_MODE);
+
+export const resetDisplayOptionsDispatcher = dispatch => () => {
+  dispatch(setTransliterationLanguages(DEFAULT_TRANSLITERATION_LANGUAGES));
+  dispatch(setTranslationLanguages(DEFAULT_TRANSLATION_LANGUAGES));
+  dispatch(setDarkMode(DEFAULT_DARK_MODE));
+}
+
+export const resetFontOptionsDispatcher = dispatch => () => {
```

Comment:
You need to accept payload first and then (dispatch, getState). For that to work we'll need [redux-thunk](https://github.com/reduxjs/redux-thunk/blob/master/README.md). Sorry I forgot to mention that earlier.